src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        int index = random.nextInt(colors.length);
+        return String.valueOf(colors[index]);
```

Comment:
Instead of using `String.valueOf()`, you should use the `name()` method to get the String representation of the enum constant. This ensures that even if the `toString()` method is overridden in the enum, you will get the correct name of the constant.